simple-service-webapp-tests/src/test/java/com/example/UserStoriesRestTest.java

```diff
@@ -166,6 +166,7 @@ public void testDeleteMethod() throws IOException, ParseException {
                         .setHeaders(new HttpHeaders()
                                 .set("Authorization", Collections.singletonList(stringToken))
                         );
+        exception.expect(HttpResponseException.class);
```

Comment:
That's an interesting approach, I did not know the google http client had that, but in this way you don't know the exeception details, only that it is an exceptions. Why not just put a try catch around the execute, and then analyze the caught exceptions, to make sure it's a 404. Under execute, you can do assert.fail() since it was never supposed to get there